sass/routes/_loading.scss

```diff
@@ -37,15 +37,5 @@
                 @include animation-delay(.2s*$i);
             }
         }
-
-        @include keyframes(move) {
```

Comment:
This was failing in IE11 when running cssVars function, if I remove it it works perfectly fine. I still have to find the solution for that.

Reply:
Finally I decide to move it out to the root of the file, which i think is the proper solution, because makes no sense to include it as a child in the css.